src/main/java/org/gridsuite/study/server/NetworkModificationTreeService.java

```diff
@@ -292,6 +343,22 @@ public String getUniqueNodeName(UUID studyUuid) {
         return uniqueName;
     }
 
+    @Transactional(readOnly = true)
+    public String getSuffixedNodeName(UUID studyUuid, String nodeName) {
+        List<String> studyNodeNames = networkModificationNodeInfoRepository.findAllByNodeStudyId(studyUuid)
```

Comment:
It's not your PR but this "findAllByNodeStudyId is weird nope ?

Reply:
I agree that the name is VERY weird and i'd be ok to change it. Are you ok with that @jonenst

Reply:
it's the official way to do it in spring data jpa: https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.query-methods.query-property-expressions